charts/sonarqube/README.md

```diff
@@ -29,6 +29,14 @@ The above command deploys SonarQube on the Kubernetes cluster in the default con
 
 The default login is admin/admin.
 
+## Installing the LTS chart
+
+The LTS chart is being distributed as the `8.x.x` version of this chart and it is compatible with the current SonarQube LTS (`9.9.x`).
+
+In order to use it, please set the version constraint `~8`, which is equivalent to `>=8.0.0 && <= 9.0.0`. That version parameter **must** be used in every helm related command including `install`, `upgrade`, `template`, and `diff` (don't treat this as an exhaustive list).
```

Comment:
Similar as for sonarqube-dce: does it make sense to provide an example?

Reply:
I think it is worth mentioning.

the helm workflow is so integrated that it is not to be assumed that people knows how the repository update works and how to handle such cases.

Waiting for your opinion on that

Reply:
I think it's great to mention how to handle the LTS case. My question was in fact a suggestion to add more detailed explanations if we can.

Reply:
hoo i see, understood it the other way sorry...

That section is like complementary to the `install the chart` section above, but yes we could add a more precise example with something like this ?

```
helm upgrade --install -n sonarqube --version ~8 sonarqube sonarqube/sonarqube
```

Reply:
👍